test/asciidoctor_test.rb

```diff
@@ -1,4 +1,4 @@
-require File.expand_path('helper', __dir__)
+require 'test_helper'
```

Comment:
Should we do `require_relative 'test_helper'` instead?

Reply:
I used `require` because it's the one I've found in other projects ([Rails](https://github.com/rails/rails/blob/main/actiontext/test/integration/controller_render_test.rb#L3), [Pay](https://github.com/pay-rails/pay/blob/master/test/models/pay/subscription_test.rb#L1)) and because is what rubygems use when creating a new gem.

The only issue I could think of is that running tests manually from inside the `test` folder (or any other place different from the root path) won't work, as it won't find the file.

Do you foresee any other issue by not using `require_relative`?

Reply:
Those were the issues I was thinking about. The code before could handle those scenarios, right?

I also recall reading relative being faster.

Reply:
See also https://github.com/puma/puma/issues/2902, https://github.com/puma/puma/pull/2964

Reply:
Enough arguments :smile:  updated to use `require_relative` instead